styles/styles.css

```diff
@@ -0,0 +1,692 @@
+@import url("https://fonts.googleapis.com/css?family=Heebo:400:700&display=swap");
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+article,
+div {
+  min-height: 10px;
+}
+
+h2.tnw-section {
+  color: #d63a00;
+  font-family: 'Heebo', sans-serif;
+  font-size: 30px;
+  font-weight: bold;
+  margin: 3em 0px .7em 0px;
+}
+
+#grid-container {
+  margin: auto;
+  grid-template-areas: "top-menu" "main" "footer";
+  grid-template-columns: auto;
+  grid-template-rows: repeat(3, auto);
+}
+
+/*nav*/
+#whole-nav {
+  display: grid;
+  grid-template-areas: "top-nav" "bottom-nav" "icon";
+  grid-template-columns: 1fr;
+  grid-template-rows: 32px 48px;
+}
+#whole-nav #top-nav {
+  grid-area: top-nav;
+  background-color: #131313;
+  color: gray;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+}
+#whole-nav #top-nav a {
+  color: gray;
+  font-weight: bold;
+}
+#whole-nav #bottom-nav {
+  grid-area: bottom-nav;
+  color: gray;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+#whole-nav #bottom-nav a {
+  color: gray;
+  font-weight: bold;
+}
+#whole-nav #icon {
+  grid-area: icon;
+  width: 0px;
+  height: 0px;
+  margin: 0;
+}
+
+@media screen and (max-width: 1024px) {
+  #whole-nav {
+    display: grid;
+    grid-template-areas: "icon top-nav" "icon bottom-nav";
```

Comment:
For more readability, let's put all styles that related to `1024px` breakpoint into the same block, also for `768px`